### ISSUE TITLE
[exam][speaking test]2023-04

### ISSUE THREAD

**Issue by coffee-qm** [coffee-qm/toeic#4]
[toc]

##### Text 1

Are you worried about your valuables getting damaged when you travel? Then worry no more because now you can purchase the Mega Bag. The Mega Bag is a durable and waterproof travel case that protects your jewelry, mobile phone, and other important items. What's more, it comes in a variety of colors. Buy your Mega Bag today!

##### Text 2

Ladies and gentleman, welcome to tonight's performance featuring the city's best actors, musicians, and dancers. While you are watching the performance, please keep in mind that taking pictures is not allowed. In addition, we request that you keep your cell phone switched off at all times. Thank you for your cooperation.

##### Picture 1

<img width="814" alt="TOEIC_SpeakingTest_2023-04_part2_1" src="https://github.com/coffee-qm/toeic/assets/32775335/f8afada3-53cb-4f68-aa06-4323d3acf703">

This is the picture taken outdoors, on the street. In the middle, there are two men walking side by side. 
On the left, the man with a beard is wearing a hat and a brown jacket. He is making a gesture while talking. 
On the right, the other man is looking at him and smiling. It seems that he is listening to the story his friend is telling. He is holding a mobile phone in his hand.
On the right, I can see some scooters parked along the street.
In the background, there are many cars moving on the road.

##### Picture 2

<img width="825" alt="TOEIC_SpeakingTest_2023-04_part2_2" src="https://github.com/coffee-qm/toeic/assets/32775335/dc0421fa-2be5-4d23-baeb-e91f21cd3b03">

This is the picture taken at the lobby of an airport. There are many people in the picture.
On the right, there are two old people, one man and one woman, standing next to each other. The woman wearing a jean shirt is holding her suitcase. The man wearing glasses is looking at his passport. Beside him, I can see a cart with some luggage stacked on it.
On the left, there are some people standing and holding their luggage.
Overall, people are waiting for their check-in procedure.

##### Topic: Purchase textbooks

Imagine that a local magazine is conducting a survey on purchasing textbooks. You have agreed to join a telephone interview regarding buying books.

Q: Where is the ideal place to buy textbooks in your area? Have you ever bought books there?
A: The ideal site to purchase textbooks in my neighborhood is a store named Sky Bookstore. It offers a wide range of textbooks at reasonable prices.

Q: Do you think it is a good idea to buy books through online platforms?
A: Yes, that is a good idea to get books through the Internet. It is quite convenient because you can sit at one place and browse many stores to choose the books you prefer.

Q: Do you like reading electronic books or paper ones? Why?
A: 
Personally, I prefer reading books on electronic devices over paper books.
First of all, it is convenient for me when I need to travel somewhere. I do not need to carry heavy physical books with me.
In addition, buying electronic versions can help me save money. For example, currently, I read books through the Kindle app on my iPad. The price to get the Kindle version is much cheaper than the paper one.
Also, after the payment process, I can enjoy the book immediately. Unlike paper books, I need to wait for delivery.
Therefore, I am in favor of reading materials on the electronic devices.

##### Topic: Magnificent Moment Event Planner

<img width="934" alt="TOEIC_SpeakingTest_2023-04_part4" src="https://github.com/coffee-qm/toeic/assets/32775335/7ed64479-841f-4fbf-9a30-05de41a58312">

Hello, this is Sarah. I misplaced my schedule for May 23rd, so can you answer some questions for me to confirm a few things?

Q: When does the first meeting start and finish? And who will I meet?
A: Hello, your first meeting will begin at eight thirty A.M. and end at nine thirty A.M. You will have an appointment with Springhill's banquet manager, Haley Taylor.

Q: I remember that I will have lunch at noon with Andrew Simmons. Is this right?
A: I'm afraid that your information is not correct. At noon, you will have lunch with Beth Bryant, the event photographer.

Q: It's supposed that there are some sessions after lunch. Can you give me details of the activiies I need to do in the afternoon?
A: Well, according to the schedule, you will attend two activities after having lunch with Beth Bryant. From one P.M. to three P.M., you will meet Andrew Simmons from Pure Catering Kitchen to talk about the menu prices of the event. After that, from three P.M. to four P.M., you will stop by Abigail's Flower Creations to take sample decorations and catalogues.

##### Topic: Art course

Q: Do you think that current employees should learn one art course while working at the company?
A: 
In my opinion, individuals should take part in one art course while working at one company because of the following reasons.
First of all, the art course can help employees stimulate their creativity. For example, if people are working in creative industries such as designers or artists, this class is essential for them to gain more knowledge. This assists them to create innovative works while designing artworks like posters, banners, or advertising videos. Therefore, employee performance will improve.
In addition, art courses can also help workers relieve stress and be seen as a hobby after work. They can help employees forget all the tiredness and refresh their mind. Hence, individuals can better their duties at work.
In conclusion, I am favor of letting people participate in art classes. These courses can develop their creative ability and can be supposed to be an activity to release stress.
